x-pack/legacy/plugins/reporting/export_types/csv_from_savedobject/types.d.ts

```diff
@@ -25,9 +25,10 @@ export interface JobParamsPanelCsv {
   visType?: string;
 }
 
-export interface JobDocPayloadPanelCsv extends JobDocPayload {
+export interface JobDocPayloadPanelCsv extends JobDocPayload<JobParamsPanelCsv> {
   type: string | null;
   jobParams: JobParamsPanelCsv;
+  objects?: null;
```

Comment:
So `objects` can either be omitted or `null`? No `object` type?
It feels like something is amis here, but I lack context... perhaps a comment explaining why `objects` is expected to be `null`?

Reply:
171f7ede64